src/d/actor/d_a_bomb2.cpp

```diff
@@ -618,17 +618,17 @@ namespace daBomb2 {
     }
 
     void Act_c::set_nut_exp_interval() {
-        if(field_0x738 > attr().field_0xA) {
-            field_0x738 = attr().field_0xA;
+        if(mBombTimer > attr().field_0xA) { // attr().field_0xA appears to be the bomb timer cap
+            mBombTimer = attr().field_0xA;
 
             f32 frame = 0x87 - attr().field_0xA;
             mBck0.getFrameCtrl()->setFrame(frame);
             mBrk0.getFrameCtrl()->setFrame(frame);
         }
     }
-
+    // runs mBrk0.play twice if the timer is less than 0x87
```

Comment:
This comment doesn't provide much value as it simply states the function's function.